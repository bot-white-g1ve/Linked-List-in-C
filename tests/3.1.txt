NEW 1
a
NEW 2
{0}
b
VIEW 1
VIEW-NESTED 1
VIEW ALL
INSERT 1 0 {0}
VIEW 1
VIEW-NESTED 1