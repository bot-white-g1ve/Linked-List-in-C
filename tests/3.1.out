List 0: a
Nested 1: {List 0} -> b
{List 0} -> b
{a} -> b
Number of lists: 2
List 0
Nested 1
Nested 1: {List 0} -> {List 0} -> b
{List 0} -> {List 0} -> b
{a} -> {a} -> b
